### ISSUE TITLE
Move to BepInEx to Support Subnautica 2.0

### ISSUE THREAD

**Issue by jet082** [tobeyStraitjacket/VersionChecker#30]
I know that you are aware of this already, but just to keep track of the problem and note when it is resolved, I'm posting an issue here for best practices.

**Comment by toebeann** [comment]
Yeah. The issue is bigger than simply moving to BepInEx - the way mods tell VersionChecker where to look for updates needs to be redesigned as QMM's mod.json is no longer relevant in BepInEx.

I have ideas for how to address this and it's planned to be worked on when I'm back from holidays.
